src/test/regress/expected/upgrade_citus_finish_citus_upgrade.out

```diff
@@ -0,0 +1,14 @@
+-- Citus upgrades are finished by calling a procedure
+-- this is a transactional procedure, so rollback should be fine
+BEGIN;
+	CALL citus_finish_citus_upgrade();
+NOTICE:  Preparing all the existing partitioned table indexes
+NOTICE:  Preparing to sync the metadata to all nodes
+ROLLBACK;
+-- do the actual job
+CALL citus_finish_citus_upgrade();
```

Comment:
Wondering whether we should check the pg_dist_node_metadata upgrade_version

Reply:
makes sense